pkgs/applications/audio/strawberry/default.nix

```diff
@@ -0,0 +1,45 @@
+{ mkDerivation, stdenv, lib, fetchFromGitHub, cmake, pkgconfig
+, alsaLib, boost, chromaprint, fftw, gnutls, libcdio, libmtp, libpthreadstubs, libtasn1, libXdmcp, pcre, protobuf, sqlite, taglib
```

Comment:
This line is quite long, could it be split into two?

Reply:
I've just run nixpkgs-fmt on it to clean things up.

Reply:
I wasn't aware nixpkgs-fmt was released yet.

Reply:
It's in unstable.